### PR TITLE
FD1 optical sim update

### PR DESCRIPTION
**Pull request by lpaulucc**
Changing the optical sim flow for FD1-HD from the multiopdetsim to a single version of it.

### CODE REVIEW

**Comment by lpaulucc** [comment]
The new config worked fine for detsim stage and was tested with optical reco/ana alone because I could not run reco_dune10kt_1x2x6_wirecell_refactored.fcl till the end because of a pandora shower error.

**Comment by absolution1** [comment]
Hi @lpaulucc 
What was the error?  Can you provide an example file for me to test?

File: fcl/dunefd/detsim/detsim_dune10kt_wirecell_refactored.fcl [hunk 1]
```diff
@@ -89,3 +88,5 @@ services.ChannelNoiseService.NoiseNormZ:  4.5
 
 #configs: ["pgrapher/experiment/pdsp/wcls-sim-drift-simchannel.jsonnet"]
 #=> configs: ["pgrapher/experiment/dune10kt-1x2x6/wcls-sim-drift-simchannel.jsonnet"]
+physics.producers.sipm.InputTag:  "PDFastSim"
+physics.producers.opdigi.InputTags:   [sipm]
```
Comment (absolution1) [inline fcl/dunefd/detsim/detsim_dune10kt_wirecell_refactored.fcl:92]: Is it possible to move these into the upstream workflow fcl?

**Review by absolution1 — COMMENTED** [review]
Thanks @lpaulucc 
I've included a single suggestion about moving some overrides.

**Comment by lpaulucc** [comment]
The error at reco stage was this:
```
%MSG-s ArtException:  PostEndJob 13-Jan-2023 08:37:03 CST ModuleEndJob
---- EventProcessorFailure BEGIN
  EventProcessor: an exception occurred during current event processing
  ---- ScheduleExecutionFailure BEGIN
    Path: ProcessingStopped.
    ---- OtherArt BEGIN
      ---- ShowerElementHolder BEGIN
        Trying to get Element: ShowerStartPosition. This element you are filling is not the correct type
        The above exception was thrown while processing module LArPandoraModularShowerCreation/pandoraShower run: 20000001 subRun: 0 event: 2
      ---- ShowerElementHolder END
    ---- OtherArt END
    Exception going through path reco
  ---- ScheduleExecutionFailure END
---- EventProcessorFailure END
---- FatalRootError BEGIN
  Fatal Root Error: TTree::SetEntries
  Tree branches have different numbers of entries, eg art::TriggerResults_TriggerResults__Reco. has 1 entries while EventAuxiliary has 10 entries.
  ROOT severity: 2000
---- FatalRootError END
```
The sequence of fhicls was 
prodgenie_nu_dune10kt_1x2x6.fcl
standard_g4_dune10kt_1x2x6_refactored.fcl
detsim_dune10kt_1x2x6_wirecell_refactored.fcl
reco_dune10kt_1x2x6_wirecell_refactored.fcl
and the detsim file can be found here: /dune/app/users/lpaulucc/dunesw/prodgenie_nu_dune10kt_1x2x6_gen_g4_detsim.root

**Comment by tomjunk** [comment]
Sorry to be slow with this PR.  does it depend on Dom's bug report and PR?  Or can it be merged at any time?

**Comment by lpaulucc** [comment]
Hi Tom, no problem. This PR does not depend on fixing the bug that Dom addressed in another PR. It can be merged at any time.

**Pull request merged**